app/controllers/objects_controller.rb

```diff
@@ -86,6 +86,11 @@ def content_diff
     Honeybadger.notify(e)
   end
 
+  # Retrieves the primary complete moab for a preserved objects and returns the output in JSON format
+  def primary_moab
+    render json: PreservedObject.find_by!(druid: druid).preserved_objects_primary_moab.complete_moab.to_json
+  end
```

Comment:
since pres robots will be using this to figure out the path to use for updating a specific copy of a moab, and since the storage location doesn't live directly on the `CompleteMoab` model, i think we need to return something more like:

```ruby
PreservedObject.find_by!(druid: druid).preserved_objects_primary_moab.complete_moab.moab_storage_root.storage_location
```

or maybe...

```ruby
MoabStorageRoot.joins(complete_moabs: [:preserved_object, :preserved_objects_primary_moab]).find_by!(preserved_objects: { druid: druid }).storage_location
```

i tested locally just now, and that seemed to do the right thing in one query, e.g.:
```ruby
[17] pry(main)> druid = PreservedObject.first.druid
  PreservedObject Load (2.6ms)  SELECT "preserved_objects".* FROM "preserved_objects" ORDER BY "preserved_objects"."id" ASC LIMIT $1  [["LIMIT", 1]]
=> "bj102hs9687"
[18] pry(main)> 
[26] pry(main)> MoabStorageRoot.joins(complete_moabs: [:preserved_object, :preserved_objects_primary_moab]).find_by!(preserved_objects: { druid: druid })
  MoabStorageRoot Load (3.0ms)  SELECT "moab_storage_roots".* FROM "moab_storage_roots" INNER JOIN "complete_moabs" ON "complete_moabs"."moab_storage_root_id" = "moab_storage_roots"."id" INNER JOIN "preserved_objects" ON "preserved_objects"."id" = "complete_moabs"."preserved_object_id" INNER JOIN "preserved_objects_primary_moabs" ON "preserved_objects_primary_moabs"."complete_moab_id" = "complete_moabs"."id" WHERE "preserved_objects"."druid" = $1 LIMIT $2  [["druid", "bj102hs9687"], ["LIMIT", 1]]
=> #<MoabStorageRoot:0x00007fd8bc8709c8
 id: 1,
 name: "fixture_sr1",
 created_at: Wed, 24 Jun 2020 00:58:44 UTC +00:00,
 updated_at: Wed, 24 Jun 2020 00:58:44 UTC +00:00,
 storage_location: "spec/fixtures/storage_root01/sdr2objects">
[27] pry(main)> 
[35] pry(main)> MoabStorageRoot.joins(complete_moabs: [:preserved_object, :preserved_objects_primary_moab]).find_by!(preserved_objects: { druid: druid }).storage_location
  MoabStorageRoot Load (5.0ms)  SELECT "moab_storage_roots".* FROM "moab_storage_roots" INNER JOIN "complete_moabs" ON "complete_moabs"."moab_storage_root_id" = "moab_storage_roots"."id" INNER JOIN "preserved_objects" ON "preserved_objects"."id" = "complete_moabs"."preserved_object_id" INNER JOIN "preserved_objects_primary_moabs" ON "preserved_objects_primary_moabs"."complete_moab_id" = "complete_moabs"."id" WHERE "preserved_objects"."druid" = $1 LIMIT $2  [["druid", "bj102hs9687"], ["LIMIT", 1]]
=> "spec/fixtures/storage_root01/sdr2objects"
[36] pry(main)> 
[37] pry(main)> PreservedObjectsPrimaryMoab.count
   (2.7ms)  SELECT COUNT(*) FROM "preserved_objects_primary_moabs"
=> 6
[38] pry(main)> 
[41] pry(main)> PreservedObjectsPrimaryMoab.destroy_all
  PreservedObjectsPrimaryMoab Load (4.0ms)  SELECT "preserved_objects_primary_moabs".* FROM "preserved_objects_primary_moabs"
   (2.3ms)  BEGIN
...
[46] pry(main)> PreservedObjectsPrimaryMoab.count
   (3.3ms)  SELECT COUNT(*) FROM "preserved_objects_primary_moabs"
=> 0
[47] pry(main)> 
[50] pry(main)> MoabStorageRoot.joins(complete_moabs: [:preserved_object, :preserved_objects_primary_moab]).find_by!(preserved_objects: { druid: druid }).storage_location
  MoabStorageRoot Load (2.8ms)  SELECT "moab_storage_roots".* FROM "moab_storage_roots" INNER JOIN "complete_moabs" ON "complete_moabs"."moab_storage_root_id" = "moab_storage_roots"."id" INNER JOIN "preserved_objects" ON "preserved_objects"."id" = "complete_moabs"."preserved_object_id" INNER JOIN "preserved_objects_primary_moabs" ON "preserved_objects_primary_moabs"."complete_moab_id" = "complete_moabs"."id" WHERE "preserved_objects"."druid" = $1 LIMIT $2  [["druid", "bj102hs9687"], ["LIMIT", 1]]
ActiveRecord::RecordNotFound: Couldn't find MoabStorageRoot with [WHERE "preserved_objects"."druid" = $1]
from /Users/suntzu/.rbenv/versions/2.7.1/lib/ruby/gems/2.7.0/gems/activerecord-6.0.3.2/lib/active_record/relation/finder_methods.rb:336:in `raise_record_not_found_exception!'
```